tests/CMakeLists.txt

```
@@ -151,3 +151,11 @@ endif()
 
 string(CONCAT t ${OS_TEST_CMDS})
 message("\nTo run unit tests, use: ${t}")
+
+if (ENABLE_COVERAGE)
+  setup_target_for_coverage_lcov(
+    NAME test_coverage
```

```suggestion
    NAME coverage
```

What do you think about just calling it 'coverage', so you can build the report with '`make coverage`'? It matches the `ENABLE_COVERAGE` option name, and it's slightly more concise.